src/pages/workspaces/List.js

```diff
@@ -262,14 +268,17 @@ export const WorkspaceList = () => {
             const lastRunStatus = workspaceSubmissionStatus(workspace)
 
 
-            return div({ style: { ...styles.tableCellContainer, paddingRight: 0 } }, [
+            return canView ? div({ style: { ...styles.tableCellContainer, paddingRight: 0 } }, [
```

Comment:
I decided to remove the button to popup the menu unless the user can actually access it. There is much discussion online about how disabled buttons do not provide a good user experience since it isn't obvious how a user might go about enabling them. In this particular case, the menu button itself provides no meaning if the user can't interact with the workspace anyway

Reply:
Instead of a ternary, you can just do `canView && div(...`